spec/javascripts/reducers/column_backlog_spec.js

```diff
@@ -63,14 +81,49 @@ describe('Backlog Column reducer', () => {
         startDate: "2018-09-03T16:00:00",
         iterationLength: 1,
         defaultVelocity: 2,
-      },
+      }
     }
-  }
+  };
+
+  function toggleStoryAction(id) {
```

Comment:
You can import it from here https://github.com/Codeminer42/cm42-central/pull/417/files#diff-a70c128fa112ed9628c46ebccc4a8fd2R22